workspace/Gemfile

```diff
@@ -6,5 +6,5 @@ gem 'git'
 gem 'lita'
 gem 'lita-slack'
 gem 'nokogiri'
-
+gem 'activesupport'
```

Comment:
why do we need this gem?  Time should already be accessible?!

Reply:
@scopp I need Active Support for PST timezone since in Time we can only access timezone in UTC